assay/skills/critique/references/dependency-patterns.md

```
@@ -0,0 +1,319 @@
+**How to measure:**
+```text
+For module M:
+  Ce(M) = count of unique modules that M imports from
+```
+
+### Instability (I)
+
+**Definition:** `I = Ce / (Ca + Ce)` where 0 means maximally stable and 1 means maximally unstable.
+
+**Interpretation:**
+| I value | Meaning | Expectation |
```

markdownlint-cli2 (0.21.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)